docker-compose.yml

```diff
@@ -10,6 +10,18 @@ networks:
 
 services:
 
+  foundry:
+    image: ghcr.io/foundry-rs/foundry:latest
+    labels:
+      com.centurylinklabs.watchtower.enable: '${WATCHTOWER_ENABLED:-false}'
+    entrypoint: sh
+    command:
+      - '/opt/run_anvil.sh'
```

Comment:
Can we have the flags directly here? instead of the script? easier to read.

```
command:
      - --port=8540
      - --chain-id=...
      ...
```

nwaku instances use the script `run_nwaku.sh` because of the `IP`, since you don't know it beforehand, so with the script (that is executed once inside the container) we can get the IP and then configure it.

So unless there is a strong reason for having the flags in a separate script, i would prefer having them directly there.